e2e-tests/playwright.config.ts

```diff
@@ -1,7 +1,7 @@
-import { type PlaywrightTestConfig, devices } from '@playwright/test';
+import { type PlaywrightTestConfig, devices } from "@playwright/test";
 
 const config: PlaywrightTestConfig = {
-  testDir: 'src',
+  testDir: "src",
```

Comment:
Format changed because you run the formatter or because you did not run the formatter before submitting the PR?

Just asking in case useful.

Reply:
Because this file was never formatted.